eth/tracers/testdata/call_tracer_big_slow.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Gosh, how it warms my heart to see these tests! :pray:

Reply:
How did you generate these? Worth considering adding the generation program to the source somehow?

Reply:
It's very easy. You can see the actual script [here](https://github.com/etclabscore/core-geth/pull/255/files/b194b2c4f83ae3f15927accec95e01bc391e68fe#diff-6b1dc15800ea95ec572b25668c1a2f58ebae907d561623718f54e6a6cb7977e7R47-R93)

Reply:
Oh, I see... nice.

```
// To generate a new callTracer test, copy paste the makeTest method below into
// a Geth console and call it with a transaction hash you which to export.
```
```
// makeTest generates a callTracer test by running a prestate reassembled and a
// call trace run, assembling all the gathered information into a test case.
```